parm/use_cases/met_tool_wrapper/MADIS2NC/MADIS2NC.conf

```diff
@@ -0,0 +1,88 @@
+[config]
+
+# Documentation for this use case can be found at
+# https://metplus.readthedocs.io/en/latest/generated/met_tool_wrapper/MADIS2NC/MADIS2NC.html
+
+# For additional information, please see the METplus Users Guide.
+# https://metplus.readthedocs.io/en/latest/Users_Guide
+
+###
+# Processes to run
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#process-list
+###
+
+PROCESS_LIST = MADIS2NC
+
+
+###
+# Time Info
+# LOOP_BY options are INIT, VALID, RETRO, and REALTIME
+# If set to INIT or RETRO:
+#   INIT_TIME_FMT, INIT_BEG, INIT_END, and INIT_INCREMENT must also be set
+# If set to VALID or REALTIME:
+#   VALID_TIME_FMT, VALID_BEG, VALID_END, and VALID_INCREMENT must also be set
+# LEAD_SEQ is the list of forecast leads to process
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#timing-control
+###
+
+LOOP_BY = INIT
+INIT_TIME_FMT = %Y%m%d%H
+INIT_BEG = 2012040912
+INIT_END = 2012040912
+INIT_INCREMENT = 1H
+
+LEAD_SEQ = 0
+
+MADIS2NC_RUNTIME_FREQ = RUN_ONCE_FOR_EACH
+
+
+###
+# File I/O
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#directory-and-filename-template-info
+###
+
+MADIS2NC_INPUT_DIR =
+MADIS2NC_INPUT_TEMPLATE = {INPUT_BASE}/met_test/data/sample_obs/madis/metar/metar_{init?fmt=%Y%m%d%H}_F{lead?fmt=%3H}.nc
+
+MADIS2NC_OUTPUT_DIR =
+MADIS2NC_OUTPUT_TEMPLATE = {OUTPUT_BASE}/madis2nc/met_metar_{init?fmt=%Y%m%d%H}_F{lead?fmt=%3H}.nc
+
+#MADIS2NC_SKIP_IF_OUTPUT_EXISTS = False
+
+#MADIS2NC_FILE_WINDOW_BEGIN = 0
+#MADIS2NC_FILE_WINDOW_END = 0
+
+###
+# MADIS2NC Settings
+# https://metplus.readthedocs.io/en/latest/Users_Guide/wrappers.html#madis2nc
+###
+
+#LOG_MADIS2NC_VERBOSITY = 3
+
+MADIS2NC_CONFIG_FILE = {PARM_BASE}/met_config/Madis2NcConfig_wrapped
+
+MADIS2NC_TYPE = metar
+
+#MADIS2NC_QC_DD =
+#MADIS2NC_LVL_DIM =
+#MADIS2NC_REC_BEG =
+#MADIS2NC_REC_END =
+#MADIS2NC_MASK_GRID =
+#MADIS2NC_MASK_POLY =
+#MADIS2NC_MASK_SID =
+
+#MADIS2NC_TIME_SUMMARY_FLAG =
+#MADIS2NC_TIME_SUMMARY_RAW_DATA =
+#MADIS2NC_TIME_SUMMARY_BEG =
+#MADIS2NC_TIME_SUMMARY_END =
+#MADIS2NC_TIME_SUMMARY_STEP =
+#MADIS2NC_TIME_SUMMARY_WIDTH =
+#MADIS2NC_TIME_SUMMARY_GRIB_CODE =
```

Comment:
I'd really like to eliminate references to `grib_code` and instead switch fully to processing observation by their name. However, I see that our usage of `time_summary.grib_code` is spread across many tools. So having it in the time summary config option is consistent. I just don't think that it would ever have any impact on the processing of madis observations.